client/model.go

```diff
@@ -149,7 +149,11 @@ type SshKey struct {
 }
 
 type SshKeyCreatePayload struct {
-	Name           string `json:"name"`
+	Name  string `json:"name"`
+	Value string `json:"value"`
+}
+
+type SshKeyCreatePayloadExtended struct {
+	SshKeyCreatePayload
```

Comment:
Why the "extended"?  
Can we make it more similar to the other types?

Reply:
I will be glad for a better name. 
We just need separated modes for Public API used by our users, and an Internal model for our internal REST api.

Reply:
Why bother separating the two?  
How this entity different than say, Project?

Reply:
We should do it for all the models contains organizationId. as discussed with @yaronya.
we want to expose fields that can be used by the user. it is weird to expose `organizationId` and do an unnecessary validation on our side - and throw an exception to the user. just do not expose it.

Reply:
45896c3